setup.sh

```diff
@@ -356,6 +356,13 @@ check_version wksctl $WKSCTL_VERSION
 log "Creating footloose manifest"
 jk generate -f config.yaml setup.js
 
+cluster_key="cluster-key"
+if [ ! -f "$cluster_key" ]; then
+    # Create the cluster ssh key with the user credentials.
+    log "Creating SSH key"
+    ssh-keygen -q -t rsa -b 4096 -C firekube@footloose.mail -f $cluster_key -N ""
```

Comment:
I'm not familiar with the behaviour of footloose. 
Does this mean if `cluster_key` is there, footloose won't create another one to replace it?

Reply:
That's right, `footloose` will not create the key if it already exist.